specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+            "description": "Error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Snapshot_Download_ComplianceReport": {
+            "$ref": "./examples/Snapshot_ComplianceReport_Download.json"
+          },
+          "Snapshot_Download_CompliancePdfReport": {
+            "$ref": "./examples/Snapshot_CompliancePdfReport_Download.json"
+          },
+          "Snapshot_Download_ComplianceDetailedPdfReport": {
+            "$ref": "./examples/Snapshot_ComplianceDetailedPdfReport_Download.json"
+          },
+          "Snapshot_Download_ResourceList": {
+            "$ref": "./examples/Snapshot_ResourceList_Download.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OverviewStatus": {
+      "description": "The overview of the compliance result for one report.",
+      "type": "object",
+      "properties": {
+        "passedCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The count of all passed full automation control."
+        },
+        "failedCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The count of all failed full automation control."
+        },
+        "manualCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The count of all manual control."
+        }
+      }
+    },
+    "ReportComplianceStatus": {
+      "type": "object",
+      "readOnly": true,
+      "description": "A list which includes all the compliance result for one report.",
+      "properties": {
+        "m365": {
+          "description": "The Microsoft 365 certification name.",
+          "$ref": "#/definitions/OverviewStatus"
+        }
+      }
+    },
+    "ResourceMetadata": {
+      "type": "object",
+      "description": "Single resource Id's metadata.",
+      "required": [
+        "resourceId"
+      ],
+      "properties": {
+        "resourceId": {
+          "description": "Resource Id.",
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Resource type.",
+          "type": "string"
+        },
+        "resourceKind": {
+          "description": "Resource kind.",
+          "type": "string"
+        },
+        "resourceName": {
+          "description": "Resource name.",
+          "type": "string"
+        },
+        "tags": {
+          "type": "object",
+          "description": "Resource's tag type.",
+          "additionalProperties": {
+            "description": "Tag properties.",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ReportProperties": {
```

Comment:
>ReportProperties [](http://example.com/codeflow?start=5&length=16)

if any of these are not alterable using PATCH, use x-ms-mutability

Reply:
Hi Tim,
Temporarily the PUT and PATCH alterable scopes are the same.
So we only use "readonly" to mark unmodifiable variables.
Thanks for the comments, we will reconsider the specs in next version.